### PR TITLE
Mobile menu section

### PR DESCRIPTION
**Pull request by Bettenoch**
In this section I made the following changes:
    ♻Added popup.js 
    ♻Added onClick functionality on the hamburger menu for mobile version
    ♻Added onClick functionality on the navlist items for more user experience.
    ♻Restyled mobile menu section

Thank You for reviewing my portfolio!! 🙏

### CODE REVIEW

**Review by Reem-lab — COMMENTED** [review]
# Hi @Bettenoch, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌❌❌

You have submitted a project, that should be pair programming but you made it solo so please reach out to your partner to collaborate with you in the milestone and if you can't reach out send a message to the student sucess that your partner don't respond to your messges and if you are allowed to make it solor put a screentshot to the mail from student scuess ✅🔥✨

image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Gambit142 — COMMENTED** [review]
Hi @Bettenoch 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- This is a pair-programming activity and you ought to collaborate with your partner. Kindly collaborate with your partner as mentioned by the first reviewer. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -23,7 +25,7 @@
 .nav-container {
   background: var(--lighter-grey);
   width: 100%;
-  display: flex;
+  display: block;
   align-items: center;
   position: fixed;
   box-shadow: 0 1.2rem 3.2rem rgba(0, 0, 0, 0.03);
```
Comment (Qoosim) [inline style.css:31]: - [ ] It is required to stick to the Figma design as much as possible. Your overlay background is supposed to cover the whole height of the page, but this is not so on your design. Kindly ensure the overlay fills the whole page as your mobile menu items appears when the hamburger icon is clicked.

### Your Design
your-desing

### Figma Design
figma-design

- [ ] Kindly reduce the margin left for all the menu items so as to match the Figma design

### Your Design
your-space

### Figma Design
figma-space

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (Efiamotu-1) [inline index.html:1]: [OPTIONAL]
- [ ] From the figma image below you can see that the portfolio comes first before the about, but in your design you have plaved the about before the portfolio, this is inconsistent with the provided figma design please make sure to make this change it is important your design matches the figma template as much as possible. :+1: 
- [ ] I also noticed that once i click on either the `portfolio, about or contact` and i try to click the menu again it won't open until i do it a second time. this is majorly coming from your javascript implementation please try to fix these optional requiremnts befoe merging your branch. Thank you :+1: :sparkles: 
- figma design
p-1
- your design
y-1

**Review by Efiamotu-1 — APPROVED** [review]
Hi @Bettenoch , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# STATUS: APPROVED 🕺🥳 

giphy


## Highlights :dart: 
* Correct GitHub flow :heavy_check_mark: 
* Good commit messages :heavy_check_mark: 
* All linters passing :heavy_check_mark: 
* Good PR Title & summary :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, you can now Merge since it is Approved ✅, However if there are any conflicts please resolve them before merging.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**